src/backend/web/routes/feeds.js

```diff
@@ -51,4 +51,26 @@ feeds.get('/:id', async (req, res) => {
   }
 });
 
+feeds.post('/', async (req, res) => {
```

Comment:
This route should get `protect` middleware: you can't do it unless you're logged in.